lib/canvas_link_migrator/link_resolver.rb

```diff
@@ -226,6 +226,7 @@ def resolve_relative_file_url_with_qs(rel_path, qs)
             end
           end
           new_url += new_action.presence || "/preview"
+          new_url.chomp!("/")
```

Comment:
I might be a bit paranoid, but now I'm worried that we might need the slash somewhere.  Might be worth looking into a URL parser for this?  Though that might not fix the place I'm worried about it. :(

Reply:
It deleted my comment when I hit resolve. I checked down stream from this code within the gem and nothing seems to be expecting that. This is very close to the end of the process and we literally slap on the query string in the next line.

If a consuming service depends on `/?` to be in the url that would be really weird. And I'm of the opinion people long before us should have made a unit test to protect that behavior if they required it.